src/Cohesity.Powershell/Scripts/View/Add-CohesityViewWhitelist.ps1

```diff
@@ -0,0 +1,91 @@
+function Add-CohesityViewWhitelist {
+    <#
+        .SYNOPSIS
+        Add whitelist IP(s) for a given view.
+        .DESCRIPTION
+        Add whitelist IP(s) for a given view.
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Add-CohesityViewWhitelist -IP4List "1.1.1.1", "2.2.2.2" -NetmaskIP4 "255.255.255.0"
+        Add whitelist IP(s) an override global whitelist for a given view.
+        .EXAMPLE
+        Add-CohesityViewWhitelist -IP4List "1.1.1.1", "2.2.2.2" -NetmaskIP4 "255.255.255.0" -NFSRootSquash -NFSAccess "kReadWrite" -NFSAllSquash -SMBAccess "kReadWrite"
+        Add whitelist IP(s) an override global whitelist for a given view with optional parameters
+    #>
+    [OutputType('System.Collections.ArrayList')]
+    [CmdletBinding(SupportsShouldProcess = $True, ConfirmImpact = "High")]
+    Param(
+        [Parameter(Mandatory = $true)]
+        # Specifies view name.
+        [string]$ViewName,
+        [Parameter(Mandatory = $true)]
+        # Specifies IPv4 addresses.
+        [string[]]$IP4List,
```

Comment:
Do we support ipv6 , if not, should we just put this as  IPAllowlist , also these could be FQDNs
?

Reply:
Done.